test/dotcom_web/components/system_status/subway_status_test.exs

```diff
@@ -101,4 +255,45 @@ defmodule DotcomWeb.Components.SystemStatus.SubwayStatusTest do
   defp subway_status(alerts) do
     Subway.subway_status(alerts, Dotcom.Utils.DateTime.now())
   end
+
+  defp status_rows_for_alerts(alerts) do
+    subway_status =
+      alerts
+      |> Dotcom.SystemStatus.Subway.subway_status(Timex.now())
+
+    render_component(&homepage_subway_status/1, %{subway_status: subway_status})
+    |> Floki.find("a")
+  end
+
+  defp for_route(rows, route_id) do
+    rows |> Floki.find("[data-route-info=#{route_id}]")
+  end
+
+  defp route_pill_visibility_for_row(row) do
+    classes =
+      row
+      |> Floki.find("[data-route-pill]")
+      |> Floki.attribute("class")
+      |> Enum.join(" ")
+      |> String.split()
+
+    if "opacity-0" in classes do
+      :invisible
+    else
+      :visible
+    end
+  end
+
+  defp status_label_text_for_row(row) do
+    row
+    |> Floki.find("[data-status-label]")
+    |> Floki.text()
+    |> String.trim()
+    |> String.split("\n")
+    |> List.last()
+    |> String.trim()
+  end
+
+  # defp status_label_text_for_effect(:station_closure), do: "Station Closure"
+  # defp status_label_text_for_effect(effect), do: effect |> Atom.to_string() |> String.capitalize()
```

Comment:
Looks like you might want to remove this

Reply:
🧟 [Goodbye](https://github.com/mbta/dotcom/pull/2404/commits/7e93f37d04b4d86025938495b8e147968ebf007f) 🧟